authorization/src/main/kotlin/org/modelix/authorization/AuthorizationConfig.kt

```diff
@@ -198,3 +218,8 @@ private fun getBooleanFromEnv(name: String): Boolean? {
         throw IllegalArgumentException("Failed to read boolean value $name", ex)
     }
 }
+
+internal fun getVerifierForSpecificAlgorithm(algorithm: Algorithm): JWTVerifier =
+    JWT.require(algorithm)
+        .acceptLeeway(0L)
```

Comment:
[From the ](https://github.com/auth0/java-jwt/blob/fb6d00ad9773c6e7624c518feb2d06ed191287fa/lib/src/main/java/com/auth0/jwt/JWTVerifier.java#L71)

0 is actually the default, which makes sense, as that reasonably assumes synchronized clocks across devices.

Reply:
Ok, then I will remove it.

Reply:
Done